pkg/tfvars/openstack/rhcos_image.go

```diff
@@ -110,6 +73,52 @@ func uploadBaseImage(cloud string, localFilePath string, imageName string, clust
 	return nil
 }
 
+func importImage(imageName, imageID string, conn *gophercloud.ServiceClient, f *os.File) error {
```

Comment:
"imageName" is not used for anything except writing debug messages. 
It would be best to get rid of it all together. Since this function is a generic importImage just displaying the imaged ID in debug messages is good enough because the calling function will probably log what image it s trying to download. Even better just return error !=nill if a failure.